addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -340,10 +364,72 @@ const Wysiwyg = Widget.extend({
     redo: function () {
         this.odooEditor.historyRedo();
     },
+    /**
+     * Open the Alt dialog
+     * dialog give access to edit <img> alt and title
```

Comment:
I would write it with the punctuation.
Also with an emty line between the first and the second as the first is the "short description".

```
Open the Alt dialog.

Dialog give access to edit <img> alt and title.
```

Reply:
well it can sure be refactored if necessary.
It was more or less mirrored on the openMedia function.